website/docs/about/overview.md

```diff
@@ -11,28 +11,38 @@ A design system is a set of guidelines, standards, assets, and processes to help
 
 Design systems are often equated with a UI kit or component library; in reality an effective design system spans the entire lifecycle of a product through ideation to iteration and everything in between.
 
-### Architecture
+## Architecture
 
 While the scale of a design system largely depends on the needs of those consuming it, the fundamental structure and architecture remains largely the same across systems.
 
 - **Visual Language**: colors, typography, icons, imagery, content strategy
 - **Components**: component design, component development, tokens, development standards
+- **Patterns**: repetitive combinations of components and foundational styles that yield more complex functions and features
 - **Documentation**: education, issue tracking, contribution, content management, processes
 
 At HaschiCorp we structure our components, icons, and design tokens in a [monorepo](https://github.com/hashicorp/design-system) using yarn workspaces and publish a collection of libraries in [Figma](https://www.figma.com/files/team/1030156573400567478) for designers.
 
-### Why use the HashiCorp Design System?
+## Why use the HashiCorp Design System?
 
-While increasing visual and functional consistency accross products is the most outwardly notable result of design system adoption, the instrinsic value a design system brings can often result in a groundswell of benefits:
+While increasing visual and functional consistency across products is the most visibly notable result of design system adoption, the intrinsic value a design system brings can result in complimentary benefits:
 
 - Increased velocity in the design and development of new products and features
 - Dismantling of silos between departments that adopt and use the system
 - More time spent on solving complex problems, less time spent on duplicative work rebuilding small atoms and molecules
 - Components, foundations, and usage guidelines published by the the HashiCorp Design System meet or exceed WCAG 2.1 AA conformance out of the box in accordance with our [accessibility statement](/about/accessibility-statement).
 
-### Our consumers
+## Our consumers
 
-The HashiCorp Design System has been adopted at varying levels by all products within the R&D organization. We work closely with these teams at the integration and implementation level while collecting feedback to better understand consumer needs and inform our [roadmap](https://go.hashi.co/hds-rollout).
+HashiCorp develops and maintains a number of different products with varying sub-brands and approaches to the user experience. In an effort to bring these different product lines and teams towards a common visual language and single source of truth, the HashiCorp Design System was born.
+
+Adoption of the HashiCorp Design System across teams is driven by a number of high-value user personas to identify convergence across products:
```

Comment:
While this section is interesting, my initial reaction is that I would not include it. It's a lot of detail and exposes some internal ideas that I'm not sure we necessarily want or need to.